### PR TITLE
Share modal passes PHPCS

### PR DESCRIPTION
**Pull request by dnaluz**
Share modal found here templates/share-modal.php passes Wordpress and PHPCompatibilityWP standards.

### CODE REVIEW

File: templates/share-modal.php [hunk 1]
```diff
@@ -1,41 +1,60 @@
...
+ * @author  Playground Inc.
+ */
+?>
+<?php
+
+if ( ! empty( $_SERVER['REQUEST_URI'] ) && ! empty( $_SERVER['HTTP_HOST'] ) ) {
+	$request_uri = esc_url_raw( wp_unslash( $_SERVER['REQUEST_URI'] ) );
+	$http_host   = esc_url_raw( wp_unslash( $_SERVER['HTTP_HOST'] ) );
+
+	$url = isset( $_SERVER['HTTPS'] ) && 'on' === $_SERVER['HTTPS'] ? 'https' : 'http' . "://{$http_host}{$request_uri}";
```
Comment (kycapstick) [inline templates/share-modal.php:19]: This line is adding a duplicate http host to url which is breaking the Facebook share.

File: templates/share-modal.php [hunk 2]
```diff
@@ -1,41 +1,60 @@
...
+/**
+ * Share
+ *
+ * Share modal template
+ *
+ * @package WordPress
+ * @subpackage community-portal
+ * @version 1.0.0
+ * @author  Playground Inc.
+ */
```
Comment (kycapstick) [inline templates/share-modal.php:11]: Code sniffer is asking for a single space after this line.

Comment (dnaluz) [reply templates/share-modal.php:11]: Push changed

File: templates/share-modal.php (hunk 1 above)
Comment (dnaluz) [reply templates/share-modal.php]: Push changed

File: templates/share-modal.php (hunk 2 above)
Comment (dnaluz) [reply templates/share-modal.php:11]: Fixed

**Pull request merged**